### ISSUE TITLE
Region GB

### ISSUE THREAD

**Issue by chrisandy** [jdaehne/GoogleStoreLocator#8]
I have &region set to GB (have also tried UK) but when searching for example 'Boston' the map returns results based on Boston USA. If I search 'Boston UK' or 'Boston GB' it returns the correct result set. I realise this may be a limitation of Google Maps API.

**Comment by jdaehne** [comment]
I have the same problems... will try to find a solution for that...

**Comment by chrisandy** [comment]
Thank you - if I wanted to add GB to the end of the search field but hidden from view, could I hack that in somewhere for now?

**Comment by jdaehne** [comment]
You could may change the line [211](https://github.com/jdaehne/GoogleStoreLocator/blob/6d5f950346071c12cbcc5ea66e5bbbfde404f087/core/components/googlestorelocator/model/googlestorelocator/googlestorelocator.class.php#L211)
to something like:

```
$address = urlencode(!empty($_REQUEST['location']) ? $_REQUEST['location'] . ', GB' : $this->location);
```

This should do the trick for now. I will try to fix this as soon as possible.

**Comment by jdaehne** [comment]
Fixed in version 2.0.1
